tests/rest/datacube/test_datacube100.py

```diff
@@ -2744,7 +2744,36 @@ def test_print_json_file_path(con100, tmp_path, path_factory):
     assert path.read_text() == EXPECTED_JSON_EXPORT_S2_NDVI + "\n"
 
 
-def test_sar_backscatter_defaults(con100):
+def test_sar_backscatter_defaults(con100, requests_mock):
+    requests_mock.get(API_URL, json={"api_version": "1.0.0"})
+    processes = [
+        {
+            "id": "sar_backscatter",
+            "description": "Computes backscatter from SAR input",
+            "summary": "Computes backscatter from SAR input",
+            "parameters": [
+                {
+                    "default": "gamma0-terrain",
+                    "description": "Select the radiometric correction coefficient.",
+                    "name": "coefficient",
+                    "schema": [
+                        {
+                            "enum": [
+                                "beta0",
+                                "sigma0-ellipsoid",
+                                "sigma0-terrain",
+                                "gamma0-ellipsoid",
+                                "gamma0-terrain",
+                            ],
+                            "type": "string",
+                        },
+                    ],
+                },
+            ],
+            "returns": {"description": "incremented value", "schema": {"type": "integer"}},
+        }
+    ]
+    requests_mock.get(API_URL + "/processes", json={"processes": processes})
```

Comment:
I would not add these request_mock to the existing tests, because these tests have to keep working without change.

Instead, I would add new tests where a custom sar_backscatter is mocked.